cloudformation-prod.json

```diff
@@ -0,0 +1,168 @@
+      "Type": "String",
+      "ConstraintDescription" : "Eg. kharasta.com"
+    },
+    "ODKUser": {
+      "Description" : "The ODK admin account username",
+      "Type": "String",
+      "MinLength": "1"
+    },
+    "ODKPassword": {
+      "NoEcho": "true",
+      "Description" : "The ODK admin account password",
+      "Type": "String",
+      "MinLength": "8"
+    }
+  },
+  "Mappings" : {
+    "RegionMap" : {
+      "us-east-1"      : { "64" : "ami-40d28157" },
+      "us-west-1"      : { "64" : "ami-6e165d0e" },
+      "eu-west-1"      : { "64" : "ami-0d77397e" },
+      "ap-southeast-1" : { "64" : "ami-a1288ec2" },
+      "ap-northeast-1" : { "64" : "ami-0567c164" }
+    }
+  },
+
+  "Resources" : {
+    "DBSecurityGroup": {
+      "Type": "AWS::EC2::SecurityGroup",
+      "Properties" : {
+        "GroupDescription": "Open database for access",
+        "SecurityGroupIngress" : [{
+          "IpProtocol" : "tcp",
+          "FromPort" : "3306", 
+          "ToPort" : "3306",
+          "SourceSecurityGroupName" : { "Ref" : "InstanceSecurityGroup" }
+        }]
+      }
+    },
+    "MainDB" : {
+      "Type" : "AWS::RDS::DBInstance",
+      "Properties" : {
+        "DBName" : "odk_stt",
+        "AllocatedStorage" : "5",
+        "DBInstanceClass" : "db.t2.micro",
+        "Engine" : "MySQL",
+        "EngineVersion" : "5.7",
+        "MasterUsername" : { "Ref" : "DBUser" },
+        "MasterUserPassword" : { "Ref" : "DBPassword" },
+				"VPCSecurityGroups" : [{ "Fn::GetAtt": [ "DBSecurityGroup", "GroupId" ] }]
+      }
+    },
+		"Ec2Instance" : {
+      "Type" : "AWS::EC2::Instance",
+      "Properties" : {
+        "ImageId" : { "Fn::FindInMap" : [ "RegionMap", { "Ref" : "AWS::Region" }, "64"]},
+        "KeyName" : { "Ref" : "KeyName" },
+				"InstanceType" : "t2.micro",
+				"SecurityGroups" : [ { "Ref" : "InstanceSecurityGroup" } ],
+        "Tags": [{
+          "Key" : "Name",
+          "Value" : "STT server"
+        }],
+				"UserData" : { "Fn::Base64" : { "Fn::Join" : [ "", [
+					"#!/bin/bash \n",
+          "echo \"",
+          "MYSQL_USER=", { "Ref" : "DBUser"}, "\n",
+          "MYSQL_DATABASE=", "odk_stt", "\n",
+          "MYSQL_PASSWORD=", { "Ref" : "DBPassword"}, "\n",
+          "DB_CONTAINER_NAME=", { "Fn::GetAtt": [ "MainDB", "Endpoint.Address" ] }, "\n",
+          "STT_DB_USER=", { "Ref" : "DBUser"}, "\n",
+          "STT_DB_PASSWORD=", { "Ref" : "DBPassword" }, "\n",
+          "STT_DB_NAME=stt\n",
+          "STT_DB_HOST=", { "Fn::GetAtt": [ "MainDB", "Endpoint.Address" ] }, "\n",
+          "STT_DB_PORT=", { "Fn::GetAtt": [ "MainDB", "Endpoint.Port" ] }, "\n",
+          "STT_USER=", { "Ref" : "STTUser" }, "\n",
+          "STT_PASSWORD=", { "Ref" : "STTPassword" }, "\n",
+          "ODK_USERNAME=", { "Ref" : "ODKUser" }, "\n",
+          "ODK_PASSWORD=", { "Ref" : "ODKPassword" }, "\n",
+          "TL_HOSTNAME", { "Ref" : "TLDomain" }, "\n",
```

Comment:
Little typo here. It's currently producing the credentials file below:
```
MYSQL_USER=stt
MYSQL_DATABASE=odk_stt
MYSQL_PASSWORD=password
DB_CONTAINER_NAME=tm5m42wc1391km.cs6gmzatc8iz.us-east-1.rds.amazonaws.com
STT_DB_USER=stt
STT_DB_PASSWORD=password
STT_DB_NAME=stt
STT_DB_HOST=tm5m42wc1391km.cs6gmzatc8iz.us-east-1.rds.amazonaws.com
STT_DB_PORT=3306
STT_USER=stt
STT_PASSWORD=password
ODK_USERNAME=stt
ODK_PASSWORD=password
TL_HOSTNAMEkharatsa.com
```

This line should change to:

```
"TL_HOSTNAME=", { "Ref" : "TLDomain" }, "\n",
```